### PR TITLE
Feat/automations

### PR DESCRIPTION
**Pull request by builderjer**
I am not sure about all of the automatons, this will definitely need some review.

### CODE REVIEW

File: .github/workflows/publish_alpha.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.8
+      - name: Install Build Tools
+        run: |
+          python -m pip install build wheel
+      - name: Increment Version
+        run: |
+          VER=$(python setup.py --version)
+          python scripts/bump_alpha.py
```
Comment (JarbasAl) [inline .github/workflows/publish_alpha.yml:37]: you need to either move to the Neon shared actions, or add the scripts folder also

**Comment by mikejgray** [comment]
@builderjer Do you want to keep this PR open and switch over to sgee's new automations, or just close it and handle that in a separate branch?

**Comment by builderjer** [comment]
Close it will be fine. His automatons are great

**Comment by builderjer** [comment]
Closing this to allow for using the main OpenVoiceOS automations

**Pull request closed without merging**